pkg/controller/kogitoapp/kogitoapp_controller.go

```diff
@@ -125,9 +127,10 @@ var _ reconcile.Reconciler = &ReconcileKogitoApp{}
 type ReconcileKogitoApp struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client *kogitocli.Client
-	scheme *runtime.Scheme
-	cache  cachev1.Cache
+	client             *kogitocli.Client
+	scheme             *runtime.Scheme
+	cache              cachev1.Cache
+	isImagestreamReady bool
```

Comment:
With `requeue`, we don't need to store the state of the image stream, it will be checked in every iteration of the loop. 

In case the image stream is deleted, it will be created again.

Reply:
done, thanks.